lib/pundit/matchers/actions_matcher.rb

```diff
@@ -10,6 +10,8 @@ class ActionsMatcher < BaseMatcher
       ACTIONS_NOT_IMPLEMENTED_ERROR = "'%<policy>s' does not implement %<actions>s"
       # Error message when at least one action must be specified.
       ARGUMENTS_REQUIRED_ERROR = 'At least one action must be specified'
+      # Error message when exactly one action must be specified.
+      ONE_ARGUMENT_REQUIRED_ERROR = 'Exactly one action must be specified'
```

Comment:
`Only` instead of `exactly`?

Reply:
To make it sound a bit more natural, you could change the message to "Only one action may be specified."